scripts/xo-eol-repos/f18/fix_repos.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Author: Justin W. Flory <me@justinwflory.com>
+# Language: Shell
+# Filename: fix_repos.sh
+#
+# Description:
+#    Automatically fixes the One Laptop per Child (OLPC)
+#    XO laptops to use correct repository files for its
+#    version of Fedora. For the HFOSS course at the
+#    Rochester Institute of Technology, most of the XOs
+#    are running Fedora 18. This script targets setting
+#    up the EoL (end-of-life) repos for the sake of the
+#    laptop.
+#
+# License:
+#    This Source Code Form is subject to the terms of the Mozilla Public
+#    License, v. 2.0. If a copy of the MPL was not distributed with this
+#    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+# Environment variables
+KEYSIGN_DIR=/etc/pki/rpm-gpg
+WORK_DIR=$HOME/tmp
```

Comment:
So there's a `$TMPDIR` variable, which would be preferred to adding a new top-level dir in `~`.